File: c/test_test.c
#include <stdio.h>

#include "testenv.h"

int main( int argc, char** argv)
{
 
 checkComplainAndAdjustExpected( 0);

 test( 1);
 test( 1);

 checkComplainAndAdjustExpected( 2);

 test( 1);
 test( 1);
 test( 1);

 checkComplainAndAdjustExpected( 3);
 
 printf( "%s\n", result());

 return 0;
}

File: clojure1.6/interpreter/test-test.clj

(ns test-test
 (:use testenv)
)

(checkComplainAndAdjustExpected 0)
(teTest true)
(teTest true)
(checkComplainAndAdjustExpected 2)
(teTest true)
(teTest true)
(teTest true)
(checkComplainAndAdjustExpected 3)

(println (teResult))
